docs/README.md

```diff
@@ -10,9 +10,9 @@ The editor consists of several primary elements, as shown in the following figur
 
 The elements highlighted in the figure are:
 
-1. **Inserter**: A panel for inserting blocks into the content canvas
+1. **Block Inserter**: A panel for inserting blocks into the content canvas
```

Comment:
Should this be called just the "Inserter"? Using it you can insert patterns and media, as well as blocks.

Reply:
That's a good question.
At https://wordpress.org/documentation/article/wordpress-block-editor/ this section is called "Block Inserter". 
I think the reasoning is that patterns and/or media are at the end just different ways of inserting blocks.

Reply:
In my review of the previous landing page PR,[ I asked @mburridge to change it to `Inserter`](https://github.com/WordPress/gutenberg/pull/49643#discussion_r1164288783) for those same reasons. I would avoid using `Blocks` here similarly to how we are starting to say [`Templates` more instead of `Block Templates`](https://github.com/WordPress/gutenberg/pull/49792#issuecomment-1508397064).